server/s3_motr_writer.cc

```diff
@@ -699,10 +699,8 @@ void S3MotrWiter::set_up_motr_data_buffers(struct s3_motr_rw_op_context *rw_ctx,
   if (buf_idx < motr_buf_count) {
     // Allocate place_holder_for_last_unit only if its required.
     // Its required when writing last block of object.
+    unit_size_for_place_holder = 16384;
```

Comment:
I would suggest o declare a const for 16384, and use that const here.

Reply:
Will address these changes in https://jts.seagate.com/browse/EOS-20630